assets/src/components/account/AccountOverview.tsx

```diff
@@ -82,6 +82,17 @@ class AccountOverview extends React.Component<Props, State> {
     }
   };
 
+
+  sendEmailToUser = async (email:string) => {
+    try {
+      const {id: token} = await API.generateUserInvitation();
+      // Call api from elixir code
```

Comment:
@reichert621 Need help on how to call the elixir function. Should I add it in the api.ts and then call it but could not understand the flow there

Reply:
I would recommend going through https://elixir-lang.org/getting-started/introduction.html the documentation is pretty good

Reply:
Thanks I did a quick run through.I just need some understanding on should this be just a normal function call from email.ex by importing it. ?